.github/workflows/pipeline.yml

```diff
@@ -2,10 +2,11 @@ name: Elixir CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
-
+    branches: [master]
+  release:
+    types: [created]
```

Comment:
Why do we need this?

Reply:
The purpose was to add a release trigger but we already have other triggers. Maybe its better to create a new workflow file as @danfowler suggested